src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -526,20 +539,28 @@ private void ScanInterpolatedStringLiteralHoleBalancedText(char endingChar, bool
             {
                 while (true)
                 {
-                    if (IsAtEnd())
+                    char ch = _lexer.TextWindow.PeekChar();
+
+                    // See if we ran into an disallowed new line.  If so, this is recoverable, so just skip past it but
+                    // give a good message about the issue.  This will prevent a lot of cascading issues with the remainder
+                    // of the interpolated string that comes on the following lines.
+                    var allowNewLines = _isVerbatim && _allowNewlines;
+                    if (!allowNewLines && SyntaxFacts.IsNewLine(ch))
```

Comment:
Braces for consistency with the rest of the function please.

Reply:
made braces consistent.